extra/1-radio-stations.js

```diff
@@ -15,6 +15,15 @@
 
 // `getAllFrequencies` goes here
 
+function getAllFrequencies() {
+  let listFrequencies = [];
+  for (let i = 87; i <= 108; i++) {
+    listFrequencies.push(i);
+  }
+  return listFrequencies;
+}
+// console.log(getAllFrequencies());
+
```

Comment:
I am impressed that you used for loop to populate the list of frequencies array.